osbuild.spec

```diff
@@ -68,10 +68,20 @@ mkdir -p %{buildroot}%{pkgdir}/assemblers/osbuild
 %if 0%{?fc30}
 ln -s org.osbuild.fedora30 %{buildroot}%{pkgdir}/runners/org.osbuild.host
 %endif
+%if 0%{?fc31}
+ln -s org.osbuild.fedora31 %{buildroot}%{pkgdir}/runners/org.osbuild.host
+%endif
+%if 0%{?fc32}
+ln -s org.osbuild.fedora32 %{buildroot}%{pkgdir}/runners/org.osbuild.host
+%endif
 %if 0%{?el8}
 ln -s org.osbuild.rhel82 %{buildroot}%{pkgdir}/runners/org.osbuild.host
 %endif
 
+# Install Fedora 31 and Fedora 32 runners as a symlink to Fedora 30
+ln -s org.osbuild.fedora30 %{buildroot}%{pkgdir}/runners/org.osbuild.fedora31
+ln -s org.osbuild.fedora30 %{buildroot}%{pkgdir}/runners/org.osbuild.fedora32
```

Comment:
I like that those are links (until we have to change something specifically for a distro), but I prefer having them in the repository directly instead of creating them here. This way it will be more apparent what happened when we do change a symlink to an actual file.

Reply:
fixed